plantuml/LinkLoss/connection_normal.puml

```diff
@@ -0,0 +1,17 @@
+@startuml
+participant LinkLossService as LLS
+participant Gap as GAP
+participant ConnectionCompleteEvent as CCE
+
+== Link Loss Service Constructed ==
+LLS <- GAP : onConnectionComplete(event)
+activate LLS
+
+LLS -> CCE : event.getStatus()
```

Comment:
We probably don't need to model the connection completion event, a note explaining what's inside should be enough. There will be another sequence diagram for when the status has an error.